.eslintrc.json

```diff
@@ -14,11 +14,14 @@
         "ShadyCSS": false
     },
     "rules": {
-        "brace-style": [2, "1tbs", { 
-            "allowSingleLine": true 
+        "array-bracket-newline": [2, { "minItems": 2 }],
+        "array-bracket-spacing": [2, "always"],
```

Comment:
Ensure that there's always space after opening and before closing array brackets.

*Good*
```javascript
var a = [ 'foobar' ];
```

*Bad*
```javascript
var a = ['foobar'];
```